caas/kubernetes/provider/init.go

```diff
@@ -67,7 +67,7 @@ func compileK8sCloudCheckers() map[string][]k8slabels.Selector {
 			),
 			// CDK on GCE.
 			newLabelRequirements(
-				requirementParams{"juju.io/cloud", selection.Equals, []string{"gce"}},
+				requirementParams{"juju-io-cloud", selection.Equals, []string{"gce"}},
```

Comment:
Aren't we doing this an an annotation now? So it can stay as "juju.io/cloud"

Reply:
https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/7/files#diff-c199fd4624388ccceb2f04c9d66e2ef9L1152
It was changed to `juju-io-cloud` before removed, but I am not very sure what key will be used when we add it back.

Reply:
changed to `juju.io/cloud` now